src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_RootScreen/components/Announcement.tsx

```diff
@@ -5,71 +5,138 @@ import {
   getTextForLanguage,
   useTranslation,
 } from '@atb/translations';
-import {Image, View} from 'react-native';
+import {Image, Linking, View} from 'react-native';
 import {StyleSheet, useTheme} from '@atb/theme';
 import {ThemeIcon} from '@atb/components/theme-icon';
 import {Close} from '@atb/assets/svg/mono-icons/actions';
 import {PressableOpacity} from '@atb/components/pressable-opacity';
 import {insets} from '@atb/utils/insets';
+import {
+  GenericClickableSectionItem,
+  LinkSectionItem,
+  Section,
+} from '@atb/components/sections';
+import Bugsnag from '@bugsnag/react-native';
+import {AnnouncementSheet} from './AnnouncementSheet';
+import {useBottomSheet} from '@atb/components/bottom-sheet';
+import {animateNextChange} from '@atb/utils/animation';
+import {useAnalytics} from '@atb/analytics';
+import {useAnnouncementsState} from '@atb/announcements';
 
 type Props = {
   announcement: AnnouncementType;
-  onDismiss: (announcement: AnnouncementType) => void;
+  withBottomPadding: boolean;
```

Comment:
As far as I know we have stopped adding properties like this. If you need something like this at all, it should be a `style` prop that's applied to the outer element of the component.

Reply:
https://github.com/AtB-AS/kundevendt/issues/15884

Reply:
Note also that it is possible to use the `gap` style on the parent view if we want the same space between each listed announcement.

Reply:
I am going to explore this gap and fix it.

Reply:
Fixed